### PR TITLE
Put up registration information.

### PR DESCRIPTION
**Pull request by ctb**
Rendered (not automatically) at teckla.idyll.org/~t/dibsi-reg/

### CODE REVIEW

File: workshops.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+has been widely adopted across a number of science fields. While
+incredibly useful, it can also be daunting to learn. This course
+doesn't require any prior programming experience. We'll teach you the
+basics of R by writing code together and setting up our computers the
+same way you will to work on your own data after the workshop. By the
+end of the week, you'll be able to input, organize, and summarize data
+in R. You'll also learn how to visualize and present data using
+publication-quality plots and dynamic documents that combine
+descriptive writing with the results of your code.
+
```
Comment (mikoontz) [inline workshops.md]: The course will focus on laying a groundwork of basic R skills to enable future self-teaching of specific use cases. However, enrollees are encouraged to reach out to the instructor [Michael Koontz](mailto:mikoontz@gmail.com) if there are particular topics that they think would be especially valuable to cover, and we’ll try to work them into the curriculum.

File: workshops.md [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+This workshop will be a hybrid between a two-day Software Carpentry
+style workshop followed by a hack-a-thon. In this workshop
+participants will learn the fundamentals of employing reproducible
+research in the R language, followed by three days of instructor-led
+group working sessions where participants will employ the skills just
+acquired towards their own research projects.
+
+Target audience: everyone is welcome: grad students, post-docs,
+faculty, everyone. Idealy, particpants will have a working knowledge
```
Comment (mikoontz) [inline workshops.md]: I was double checking the content crossover with the other R course and saw 2 small typos. Just a heads up!

*Ideally
*participants

Comment (ctb) [inline workshops.md]: fixed! now going to spellcheck too... sigh.

File: workshops.md (hunk 1 above)
Comment (ctb) [inline workshops.md]: updated

**Pull request merged**